### PR TITLE
Updated the ResourceQuota doc for NodePort and LoadBalancer

### PR DESCRIPTION
**Pull request by yashsingh74**
Fixes : https://github.com/kubernetes/kubernetes/issues/125188

### CODE REVIEW

File: content/en/docs/concepts/policy/resource-quotas.md [hunk 1]
```diff
@@ -185,17 +185,17 @@ a poorly configured CronJob. CronJobs that create too many Jobs in a namespace c
...
+| `persistentvolumeclaims` | The total number of [PersistentVolumeClaims](/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) that can exist in the namespace.                   |
+| `pods` | The total number of Pods in a non-terminal state that can exist in the namespace.  A pod is in a terminal state if `.status.phase in (Failed, Succeeded)` is true. |
+| `replicationcontrollers` | The total number of ReplicationControllers that can exist in the namespace.                                                                                        |
+| `resourcequotas` | The total number of ResourceQuotas that can exist in the namespace.                                                                                                |
+| `services` | The total number of Services that can exist in the namespace.                                                                                                      |
+| `services.loadbalancers` | The total number of `NodePorts` allocated of Services of type `LoadBalancer` that can exist in the namespace.                                                      |
```
Comment (network-charles) [inline content/en/docs/concepts/policy/resource-quotas.md:196]: Thanks for addressing the issue, @yashsingh74!
A small suggestion:
```suggestion
| `services.loadbalancers` | The total number of `NodePorts` allocated to Services of type `LoadBalancer` that can exist in the namespace.                                                      |
```

File: content/en/docs/concepts/policy/resource-quotas.md [hunk 2]
```diff
@@ -185,17 +185,17 @@ a poorly configured CronJob. CronJobs that create too many Jobs in a namespace c
...
+| `services.nodeports` | The total number of `NodePorts` allocated of Services of type `NodePort` that can exist in the namespace.                                                                                  |
```
Comment (network-charles) [inline content/en/docs/concepts/policy/resource-quotas.md:197]:
```suggestion
| `services.nodeports` | The total number of `NodePorts` allocated to Services of type `NodePort` that can exist in the namespace.                                                                                  |
```

**Comment by yashsingh74** [comment]
cc: @aojea

**Comment by T-Lakshmi** [comment]
Thanks for fixing the issue @yashsingh74 
/lgtm

**Comment by T-Lakshmi** [comment]
[Preview Page](https://deploy-preview-46594--kubernetes-io-main-staging.netlify.app/docs/concepts/policy/resource-quotas/#object-count-quota)

File: content/en/docs/concepts/policy/resource-quotas.md [hunk 3]
```diff
@@ -185,17 +185,17 @@ a poorly configured CronJob. CronJobs that create too many Jobs in a namespace c
...
+| `pods` | The total number of Pods in a non-terminal state that can exist in the namespace.  A pod is in a terminal state if `.status.phase in (Failed, Succeeded)` is true. |
+| `replicationcontrollers` | The total number of ReplicationControllers that can exist in the namespace.                                                                                        |
+| `resourcequotas` | The total number of ResourceQuotas that can exist in the namespace.                                                                                                |
+| `services` | The total number of Services that can exist in the namespace.                                                                                                      |
+| `services.loadbalancers` | The total number of `NodePorts` allocated to Services of type `LoadBalancer` that can exist in the namespace.                                                      |
+| `services.nodeports` | The total number of `NodePorts` allocated to Services of type `NodePort` that can exist in the namespace.                                                          |
```
Comment (aojea) [inline content/en/docs/concepts/policy/resource-quotas.md:197]: this is wrong, services.loadbalancers is ok, what need to change is `services.nodeports` that should be 
```suggestion
| `services.loadbalancers` | The total number of `NodePorts` allocated to Services of type `LoadBalancer` that can exist in the namespace.                                                      |
| `services.nodeports` | The total number of `NodePorts` allocated to Services of type `NodePort` or `LoadBalancer` that can exist in the namespace.                                                          |
```

**Comment by aojea** [comment]
/hold

the fix is not right, commented there

File: content/en/docs/concepts/policy/resource-quotas.md (hunk 3 above)
Comment (yashsingh74) [reply content/en/docs/concepts/policy/resource-quotas.md]: Updated the suggested changes.

**Review by kundan2707 — COMMENTED** [review]
/lgtm
thanks

**Comment by aojea** [comment]
/lgtm
/approve
/hold cancel

Thanks

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!

/approve

**Pull request merged**